openapi-generator/src/main/kotlin/org/jellyfin/openapi/hooks/api/PlayStateServiceNameHook.kt

```diff
@@ -0,0 +1,20 @@
+package org.jellyfin.openapi.hooks.api
+
+import io.swagger.v3.oas.models.Operation
+import org.jellyfin.openapi.constants.Strings
+import org.jellyfin.openapi.hooks.ServiceNameHook
+
+class PlayStateServiceNameHook : ServiceNameHook {
+	companion object {
+		const val PLAYSTATE_BAD = "Playstate${Strings.API_SERVICE_SUFFIX}"
+		const val PLAYSTATE_GOOD = "PlayState${Strings.API_SERVICE_SUFFIX}"
+	}
+
+	override fun mapServiceNames(operation: Operation, serviceNames: Set<String>) =
```

Comment:
```suggestion
	override fun mapServiceNames(operation: Operation, serviceNames: Set<String>): Set<String> =
```

Reply:
Generator doesn't use explicit API mode so it's not really necessary here. It inherits the return type anyway.

Reply:
Still, feells a bit cleaner imo and makes the return type more obvious. But whatever you prefer :P

Reply:
I didn't do it for most code so it wouldn't make sense to only do it at this bit